build.gradle

```diff
@@ -15,7 +15,7 @@ buildscript {
         classpath 'com.github.ben-manes:gradle-versions-plugin:0.17.0'
         classpath 'com.google.gms:google-services:3.1.0'
         classpath "gradle.plugin.org.jlleitschuh.gradle:ktlint-gradle:3.0.0"
-        classpath 'io.fabric.tools:gradle:1.24.4'
+        classpath 'io.fabric.tools:gradle:1.+'
```

Comment:
I think it's better to specify the concrete version number to prevent including unexpected bug.
What do you think?

Reply:
I changed according to the official reference(https://docs.fabric.io/android/crashlytics/build-tools.html). Is it wrong?

Reply:
Not wrong! but in my opinion, this official way is not good 😅

Reply:
The latest version seems to be 1.25.1.